encord/user_client.py

```diff
@@ -685,6 +692,13 @@ def __validate_filter(self, properties_filter: Dict) -> Dict:
 
         return ret
 
+    def list_groups(self) -> Iterable[OrmGroup]:
```

Comment:
again maybe ask for Sergei's opinion, but I don't see this as being super likely to ever require real pagination. A customer with 1000+ groups? By that time I'm on Bahamas after caching out during our IPO ;)